app/models/user.rb

```diff
@@ -41,4 +41,25 @@ class User < ActiveRecord::Base
   def name
     profile.try(:name)
   end
+
+  def accessible_division_ids
+    # Todo: Confirm what other roles types to include here.
```

Comment:
Actually, thinking more about this, this code really belongs in the DivisionPolicy as a scope. We should talk more about this. This feature is not slated for the release today though so we can pick it up next week.